packages/tools/client-debugger/client-debugger-view/src/components/AudienceHistoryTable.tsx

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import {
+	tokens,
+	TableBody,
+	TableCell,
+	TableRow,
+	Table,
+	TableHeader,
+	TableHeaderCell,
+} from "@fluentui/react-components";
+import { Clock20Regular, DoorArrowLeft24Regular, Person24Regular } from "@fluentui/react-icons";
+import { FilteredAudienceHistoryData } from "./AudienceView";
+
+/**
+ * Input for {@link AudienceHistoryTable}
+ */
+export interface AudienceHistoryTableProps {
+	/**
+	 * Filtered audience data from {@link audienceHistoryDataFilter}
```

Comment:
I would recommend documenting this in terms of what it means to the component consuming it, rather than how it is generated. E.g. This represents audience data filtered to the attributes that will be displayed in the table,